src/scripting/lua_game.cc

```diff
@@ -1029,7 +1035,7 @@ int LuaPlayer::allow_workers(lua_State* L) {
       full control over the player given by **playernumber** and loosing control over the
       formerly interactive player.
 
-      :arg playernumber: An index in the :class:`array` of :attr:`~wl.bases.EditorGameBase.players`.
+      :arg playernumber: :attr:`wl.game.Player.number` of player to switch to.
```

Comment:
Shouldn't this be `wl.bases.PlayerBase.number`? Same for all below.

Reply:
True, `number` is not mentioned in doc wl.game.Player (https://www.widelands.org/documentation/autogen_wl_game/#wl.game.Player.number just goes to the top)

Reply:
It's an attribute of wl.bases: https://www.widelands.org/documentation/autogen_wl_bases/#wl.bases.PlayerBase.number